elastic-mapping-updater/src/test/resources/template7.json

```diff
@@ -11,7 +11,7 @@
         }
     },
     "mappings": {
-        "dynamic": false,
+        "dynamic": "false",
```

Comment:
Haven't seen any ".kibana*" type templates on the OpenSearch servers on the stooges. Adding the quotes here so the json deserialization doesn't fail with this error:
`{"message":"testNoIndexesMatchingTemplate","exception":"org.opensearch.client.json.UnexpectedJsonEventException: Unexpected JSON event 'VALUE_FALSE' instead of '[KEY_NAME, VALUE_STRING]'\r\n\tat org.opensearch.client.json.JsonpUtils.ensureAccepts(JsonpUtils.java:81)\r\n\tat `